tests/embedding_lookup.c

```diff
@@ -0,0 +1,29 @@
+#include <finalfusion.h>
```

Comment:
System headers, blank line, non-system headers.

Reply:
There were also some more garbage imports to be removed